src/camera/camera3d.jl

```diff
@@ -132,22 +139,41 @@ function translate_cam!(scene::Scene, cam::Camera3D, _translation::VecTypes)
     dir = eyeposition - lookat
     dir_len = norm(dir)
     cam_res = Vec2f0(widths(scene.px_area[]))
-    zoom, x, y = translation
-    zoom *= 0.1f0 * dir_len
+    z, x, y = translation
+    z *= 0.1f0 * dir_len
 
     x, y = (Vec2f0(x, y) ./ cam_res) .* dir_len
 
     dir_norm = normalize(dir)
     right = normalize(cross(dir_norm, upvector))
-    zoom_trans = dir_norm * zoom
+    z_trans = dir_norm * z
     side_trans = right * (-x) + normalize(upvector) * y
-    newpos = eyeposition + side_trans + zoom_trans
+    newpos = eyeposition + side_trans + z_trans
     cam.eyeposition[] = newpos
     cam.lookat[] = lookat + side_trans
     update_cam!(scene, cam)
     return
 end
 
+function zoomtopoint_cam!(scene, point, zoom_step)
```

Comment:
Maybe change the name to `zoom!`, so that it is easier to access and interpret?

```suggestion
function zoom!(scene, point, zoom_step)
```